File: web/german_label.py
def validate_german_label(this):
    results = []
    p = py_tf.namedNode("http://example.org/ns#germanLabel")
    s = _data.find(this, p, None)

    while (t := s.next()) != None:
        object = t.getObject()
        if not object.isLiteral() or not object.getLanguage().startswith("de"):
            results.append({
                "value": object
            })

    return results

File: web/js/germanLabel.js
function validateGermanLabel($this) {
    let results = [];
    let p = TermFactory.namedNode("http://example.org/ns#germanLabel");
    let s = $data.find($this, p, null);
    for (let t = s.next(); t; t = s.next()) {
        let object = t.getObject();
        if (!object.isLiteral() || !object.getLanguage().startsWith("de")) {
            results.push({
                value: object
            });
        }
    }
    return results;
}